### PR TITLE
indentation chaneges

### PR DESCRIPTION
**Pull request by gaureshpai**
invalid comments removed
few more changes and deletion of the unused codes

### CODE REVIEW

File: app/(user)/faculty-details/page.tsx
Comment (gaureshpai) [inline app/(user)/faculty-details/page.tsx:1]: Check @jtuluve

File: app/(user)/payslip/page.tsx
Comment (gaureshpai) [inline app/(user)/payslip/page.tsx:1]: check @jtuluve

File: app/admin/firebase/page.tsx
Comment (gaureshpai) [inline app/admin/firebase/page.tsx:1]: to be done @jtuluve

File: app/admin/layout.tsx
Comment (gaureshpai) [inline app/admin/layout.tsx:1]: check @jtuluve

File: app/api/isAdmin/route.ts
Comment (gaureshpai) [inline app/api/isAdmin/route.ts:1]: check @jtuluve

File: app/firebase.ts
Comment (gaureshpai) [inline app/firebase.ts:1]: check @jtuluve

File: app/layout.tsx
Comment (gaureshpai) [inline app/layout.tsx:1]: check @jtuluve

File: app/page.tsx
Comment (gaureshpai) [inline app/page.tsx:1]: check @jtuluve 

removed the submit old button as it was not used and also removed the consecutive function of it

File: components/UserDataContext.tsx
Comment (gaureshpai) [inline components/UserDataContext.tsx:1]: check @jtuluve

File: components/academics.tsx
Comment (gaureshpai) [inline components/academics.tsx:1]: check @jtuluve

File: components/form.tsx
Comment (gaureshpai) [inline components/form.tsx:1]: check @jtuluve

File: components/menu.tsx
Comment (gaureshpai) [inline components/menu.tsx:1]: check @jtuluve

File: components/professional.tsx
Comment (gaureshpai) [inline components/professional.tsx:1]: check @jtuluve

File: components/profile.tsx
Comment (gaureshpai) [inline components/profile.tsx:1]: check @jtuluve

**Review by gaureshpai — COMMENTED** [review]
check  the changes and also waiting for ur review @jtuluve

File: app/admin/page.tsx [hunk 1]
```diff
@@ -40,16 +41,10 @@ export default function Admin() {
...
 						<thead>
 							<tr>
-								<th scope="col">Sl no</th>
-								<th scope="col">Profile</th>
-								<th scope="col">Name</th>
-								<th scope="col">Email</th>
-								<th scope="col">Department</th>
-								<th scope="col">Designation</th>
-								<th scope="col">Actions</th>
+								<th scope="col">Sl no</th><th scope="col">Profile</th><th scope="col">Name</th><th scope="col">Email</th>
```
Comment (jtuluve) [inline app/admin/page.tsx]: Keep the th tags in different lines😂🤓

File: components/table.tsx [hunk 2]
```diff
@@ -30,7 +32,6 @@ export default function Table({
   return (
     <>
       <div className={styles["table"]}>
-        {/* Check if the "values" is array or just one object and render accordingly*/}
```
Comment (jtuluve) [inline components/table.tsx]: Keep this comment

File: lib/mongoose/interface.ts [hunk 3]
```diff
@@ -98,9 +98,9 @@ interface Publication {
   date?: Date;
   patentcert?: string,
   detailsOfInventors?: {
-    number?: number;
```
Comment (jtuluve) [inline lib/mongoose/interface.ts]: bro? It shld hv space

File: app/admin/[uid]/page.tsx [hunk 4]
```diff
@@ -28,87 +27,87 @@ const UserDetail = ({ params }) => {
 		return <div>Loading...</div>;
 	}
   
-  const handlePrint = () => {
-      const printContent = printRef.current.innerHTML
```
Comment (jtuluve) [inline app/admin/[uid]/page.tsx:32]: This needs indentation!

File: app/admin/page.tsx (hunk 1 above)
Comment (gaureshpai) [reply app/admin/page.tsx]: done

File: components/table.tsx (hunk 2 above)
Comment (gaureshpai) [reply components/table.tsx]: done

File: lib/mongoose/interface.ts (hunk 3 above)
Comment (gaureshpai) [reply lib/mongoose/interface.ts]: done

File: app/admin/[uid]/page.tsx (hunk 4 above)
Comment (gaureshpai) [reply app/admin/[uid]/page.tsx:32]: done

**Pull request merged**